Dockerfile

```diff
@@ -12,13 +12,13 @@ WORKDIR /app
 # Add Poetry config
 COPY poetry.lock pyproject.toml /app/
 
+# Copy Python scripts
+COPY ./scripts/ /app/scripts/
```

Comment:
You probably want to `.gitignore` all the `tests` directories from this copy, and instead mount them when necessary.